aidial_adapter_vertexai/chat/gemini/adapter.py

```diff
@@ -24,24 +24,38 @@
 )
 from aidial_adapter_vertexai.chat.consumer import Consumer
 from aidial_adapter_vertexai.chat.errors import UserError
-from aidial_adapter_vertexai.chat.gemini.prompt import GeminiPrompt
+from aidial_adapter_vertexai.chat.gemini.prompt.base import GeminiPrompt
+from aidial_adapter_vertexai.chat.gemini.prompt.gemini_1_0_pro import (
+    Gemini_1_0_Pro_Prompt,
+)
+from aidial_adapter_vertexai.chat.gemini.prompt.gemini_1_0_pro_vision import (
+    Gemini_1_0_Pro_Vision_Prompt,
+)
+from aidial_adapter_vertexai.chat.gemini.prompt.gemini_1_5_pro import (
+    Gemini_1_5_Pro_Prompt,
+)
+from aidial_adapter_vertexai.deployments import (
+    ChatCompletionDeployment,
+    GeminiDeployment,
+)
 from aidial_adapter_vertexai.dial_api.request import ModelParameters
 from aidial_adapter_vertexai.dial_api.storage import FileStorage
 from aidial_adapter_vertexai.dial_api.token_usage import TokenUsage
-from aidial_adapter_vertexai.utils.json import json_dumps_short, to_dict
+from aidial_adapter_vertexai.utils.json import json_dumps, json_dumps_short
 from aidial_adapter_vertexai.utils.log_config import vertex_ai_logger as log
 from aidial_adapter_vertexai.utils.timer import Timer
 from aidial_adapter_vertexai.vertex_ai import get_gemini_model, init_vertex_ai
 
-HarmCategory = gapic_content_types.HarmCategory
-HarmBlockThreshold = gapic_content_types.SafetySetting.HarmBlockThreshold
-Candidate = gapic_content_types.Candidate
+HarmCategory = generative_models.HarmCategory
+HarmBlockThreshold = generative_models.HarmBlockThreshold
+GenFinishReason = generative_models.FinishReason
 
-BLOCK_NONE_SAFETY_SETTINGS: Dict[HarmCategory, HarmBlockThreshold] = {
-    HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT: HarmBlockThreshold.BLOCK_NONE,
-    HarmCategory.HARM_CATEGORY_HATE_SPEECH: HarmBlockThreshold.BLOCK_NONE,
-    HarmCategory.HARM_CATEGORY_HARASSMENT: HarmBlockThreshold.BLOCK_NONE,
-    HarmCategory.HARM_CATEGORY_SEXUALLY_EXPLICIT: HarmBlockThreshold.BLOCK_NONE,
+default_safety_settings: Dict[HarmCategory, HarmBlockThreshold] = {
+    HarmCategory.HARM_CATEGORY_UNSPECIFIED: HarmBlockThreshold.BLOCK_ONLY_HIGH,
```

Comment:
Why not NONE?

Reply:
For some reason it `BLOCK_NONE` doesn't work for certain use cases.

I have an audio-file, which triggers `400 Request contains an invalid argument` error when I use `BLOCK_NONE`.